lib/wallet/widgets/connect_wallet_button.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:ax_dapp/wallet/widgets/widgets.dart';
+import 'package:ax_dapp/wallet/widgets/dialogs/wallet_dialog.dart';
```

Comment:
This should just import widgets folder since it holds all the exports for the components including those from the dialogs folder. I could be wrong though